src/drawing/__tests__/DrawingCreator-test.tsx

```diff
@@ -129,6 +129,68 @@ describe('DrawingCreator', () => {
                 stroke: defaultStroke,
             });
         });
+
+        test.each`
+            x      | y      | expectedX | expectedY
+            ${-1}  | ${-1}  | ${4}      | ${4}
+            ${110} | ${110} | ${96}     | ${96}
+        `(
+            'should bound the drawn points ($x, $y) by the boundary of the PointerCapture element',
+            ({ x, y, expectedX, expectedY }) => {
+                const onStop = jest.fn();
+                const wrapper = getWrapper({ onStop });
+
+                simulateDrawStart(wrapper.find(PointerCapture));
+                wrapper.update();
```

Comment:
Not directly related, but we can move `wrapper.update()` to the helper to cut down on some duplication in these tests.